other-docs/guides/migrating/from-hm-platform.md

```diff
@@ -158,18 +158,6 @@ composer require --dev altis/local-server
 
 To start the docker server run `composer serve`. You should now be able to see the site at https://my-project.altis.dev where "my-project" is the project directory name.
 
-### Docker alternative
```

Comment:
Does this need to mention Codespaces now? As an altrernative to local development?

Reply:
Added a note about Codespaces in both the migration and getting started guides.

* https://github.com/humanmade/altis-documentation/pull/396/commits/c09309f90823f888047a85d407c08fadc73035f6
* https://github.com/humanmade/altis-documentation/pull/396/commits/7fc9c2a832179c12b0a5bdf2ba6e8b7a9fa890c7